.github/workflows/maintenance-desktop-audit.yml

```
@@ -179,10 +173,10 @@ jobs:
             desktops: weekly matrix audit fixes (bot)
 
             Generated by .github/workflows/maintenance-desktop-audit.yml
-            via Claude API based on the deterministic findings of
+            via Claude Code based on the deterministic findings of
             tools/modules/desktops/github/audit.py.
           title: "desktops: weekly matrix audit fixes (bot)"
-          body-path: ${{ github.workspace }}/audit-summary.md
+          body: ${{ steps.claude.outputs.conclusion }}
```

actionlint (1.7.12)

[error] property "conclusion" is not defined in object type {branch_name: string; execution_file: string; github_token: string; session_id: string; structured_output: string}

(expression)